city_scrapers/spiders/chi_ohare_noise.py

```diff
@@ -0,0 +1,215 @@
+from city_scrapers_core import constants
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+from scrapy.selector import Selector
+import requests
```

Comment:
Looks like you're getting an error because `isort` is failing, but if you run that locally it should fix it